fr.dwightstudio.JArmEmu.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id" : "fr.dwightstudio.JArmEmu",
```

Comment:
dwightstudio.fr doesn't seem to resolve

Reply:
but you say use have `dwight-studio.fr` so why not use that `fr.dwight_studio.JArmEmu` ?

(it doesn't resolve so it brings more questions)

Reply:
I own dwight-studio.fr, but the website is no longer available (until it is published to dwightstudio.fr).
If you want a proof, our private Pterodactyl server is accessible to host.dwight-studio.fr.
If it is really a problem, I can temporally change the AppID

Reply:
there is no temporarily changing app-id.

changing app-id involve resubmitting the application.

Reply:
then this will have to wait until it is sorted out.